Tiltfile

```diff
@@ -19,10 +12,16 @@ local_resource(
 
 k8s_yaml(kustomize('testing/infra/k8s/overlays/local'))
 
-k8s_kind('Agent', image_json_path='{.spec.image}')
 k8s_kind('Kibana')
 k8s_kind('Elasticsearch')
 
+custom_build('apm-server',
```

Comment:
Should we use [`docker_build`](https://docs.tilt.dev/api.html#api.docker_build) here instead of `custom_build`?

Reply:
Ah I should've mentioend this.

We can't use docker_build because the `Dockerfile` is using the `.git` folder.
From the doc: https://docs.tilt.dev/file_changes.html#git

> When you use docker_build() in your Tiltfile, Tilt will remove .git from the Docker context.
> 
> If you need to include the .git directory in a build, you can use [custom_build](https://docs.tilt.dev/custom_build.html) instead of docker_build.

Reply:
Right, I see. We should probably stop relying on `.git` in the Docker image build, but that's something we can iterate on.